src/test/codegen/repr-transparent.rs

```diff
@@ -145,7 +145,7 @@ pub struct Rgb8 { r: u8, g: u8, b: u8 }
 pub struct Rgb8Wrap(Rgb8);
 
 // NB: closing parenthesis is missing because sometimes the argument has a name and sometimes not
-// CHECK: define i32 @test_Rgb8Wrap(i32
+// CHECK: define i24 @test_Rgb8Wrap(i32
```

Comment:
This will fail on non-x86_64 linux. Probably best to move this test into a separate file with appropriate `only-*`/`ignore-*` lines.